### DISCUSSION TITLE
I am getting "AR not tracking"  issue in my phone ( samsung A23) I have tested on other devices as well but it is showing same issue. But simultaneously it is working on some devices like xiome , and one Plus, not working on samsung devices . Why ?

### DISCUSSION THREAD

**Discussion by Amit173** [google/model-viewer#4948] [Q&A]
I am getting "AR not tracking"  issue in my phone ( samsung A23) I have tested on other devices as well but it is showing same issue. But simultaneously it is working on some devices like xiome , and one Plus, not working on samsung devices . Why ?

**Comment by elalish** [comment]
I'm not sure, but that's a WebXR issue, not something we can deal with in MV. @bialpio (Chrome WebXR dev) Any ideas? Or do you know a Samsung contact we could mention here?

**Reply by Amit173** [reply]
Thank you for the response. But Idk somehow it still works on some samsung device as well. @bialpio is it a issue with Android update? Or could be something else ? Please let me know.
